src/scythe-bidder/bidarea.tsx

```diff
@@ -123,10 +121,33 @@ const BidArea = (props: {
       ></Table.Column>
       <Table.Column
         title="Faction"
-        css={{ background: "#fff" }}
         width={72}
         dataIndex="faction"
         render={(faction) => {
+          let imageLocation = "";
```

Comment:
Can we use `let imageLocation = `./static/images/${faction}.webp`;` instead?

Reply:
I saw @nrsimonelli 's reply that it's because some of them have different extensions... Could we maybe unify them? It shouldn't be hard to convert.